Gruntfile.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (grunt) {
         // Metadata.
         meta: {
             version: {
-                eva: '3.4.1'
+                eva: '3.4.2'
```

Comment:
It looks like the versions weren't kept up to date with those in JIRA. After the latest merges, the last tag should have been `v3.5`, and this one and the following 2-3 PRs would belong to `v3.6`. Applies to other files below.